Orange/classification/softmax_regression.py

```diff
@@ -10,26 +11,40 @@ class SoftmaxRegressionLearner(Learner):
     """L2 regularized softmax regression classifier.
     Uses the L-BFGS algorithm to minimize the categorical
     cross entropy cost with L2 regularization. This model is suitable
-    when dealing with a multi-class classification problem
+    when dealing with a multi-class classification problem.
 
     When using this learner you should:
 
     - choose a suitable regularization parameter lambda\_,
-    - continuize all discrete attributes,
-    - consider appending a column of ones to the data set (intercept term),
-    - transform the data set so that the columns are on a similar scale,
     - consider using many logistic regression models (one for each
       value of the class variable) instead of softmax regression.
 
-    lambda\_ : float, optional (default=1.0)
-       the regularization parameter. Higher values of lambda\_
-       force the coefficients to be smaller
+    Parameters
+    ----------
+
+    lambda_ : float, optional (default=1.0)
+        Regularization parameter. It controls trade-off between fitting the
+        data and keeping parameters small. Higher values of lambda_ force
+        parameters to be smaller.
+
+    preprocessors : list, optional (default="[Continuize(), Impute(), RemoveNaNColumns()])
+        Preprocessors are applied to data before training or testing. Default preprocessors
+        - continuize all discrete attributes,
+        - transform the dataset so that the columns are on a similar scale,
+        - remove columns with all values as NaN
+        - replace NaN values with suitable values
+
+    fmin_args : dict, optional
+        Parameters for L-BFGS algorithm.
     """
 
     name = 'softmax'
 
-    def __init__(self, lambda_=1.0, normalize=True, preprocessors=None,
-                 **fmin_args):
+    preprocessors = [Continuize(normalize_continuous=Continuize.NormalizeBySD),
+                     Impute(),
+                     RemoveNaNColumns()]
```

Comment:
See the base class, https://github.com/biolab/orange3/blob/master/Orange/classification/base.py, in particular lines 33 in 51.